### ISSUE TITLE
feature request: configuration file

### ISSUE THREAD

**Issue by kibertod** [race604/clock-tui#25]
It would be nice to have config for defining default options, default mode or default options for different modes

**Comment by race604** [comment]
Indeed, I will try to support config file. Thanks for this suggestion!

**Comment by race604** [comment]
Supported in #31.
